src/simsopt/geo/curve.py

```diff
@@ -428,11 +492,13 @@ def __init__(self, quadpoints, gamma_pure, **kwargs):
 
         self.gammadash_pure = lambda x, q: jvp(lambda p: self.gamma_pure(x, p), (q,), (ones,))[1]
         self.gammadash_jax = jit(lambda x: self.gammadash_pure(x, points))
+        self.gammadash_impl_jax = jit(lambda x, p: self.gammadash_pure(x, p))
         self.dgammadash_by_dcoeff_jax = jit(jacfwd(self.gammadash_jax))
         self.dgammadash_by_dcoeff_vjp_jax = jit(lambda x, v: vjp(self.gammadash_jax, x)[1](v)[0])
 
         self.gammadashdash_pure = lambda x, q: jvp(lambda p: self.gammadash_pure(x, p), (q,), (ones,))[1]
         self.gammadashdash_jax = jit(lambda x: self.gammadashdash_pure(x, points))
+        self.gammadashdash_impl_jax = jit(lambda x, p: self.gammadashdash_pure(x, p))
```

Comment:
What is this line for? I don't see anywhere that gammadashdash_impl_jax is used.